paddle/fluid/inference/tests/api/CMakeLists.txt

```diff
@@ -161,10 +161,10 @@ if(WITH_MKLDNN)
   endif()
   
   if (NOT EXISTS ${INT8_DATA_DIR}/pascalvoc_data.bin)
```

Comment:
pascalvoc_data.bin->pascalvoc_val_head_100.bin?

Reply:
Hi! @luotao1, In `develop` branch CMakeList.txt, `if (NOT EXISTS ${install_dir})` is used, I discussed with @wojtuss, we agree the change `if (NOT EXISTS ${install_dir}/${data_file})` is necessary. Thanks!

Reply:
Hi, @luotao1 Since PR18077, the current `develop` branch does not download pascalvoc data any more.  In `Paddle/paddle/fluid/inference/tests/api/CMakeLists.txt`, it only checks if the directory exists not file exists. 

I am worried new PRs may fail on mobilenet_ssd unit test. Wojtuss wanted to make a PR , but he found out I was changing already, so he did not push new PR to fix. Could you please take a look ? Thanks a lot !